### DISCUSSION TITLE
Harley experience

### DISCUSSION THREAD

**Discussion by USemaniuk** [lnilya/harley#11] [General]
Hello, I tried to use Harley to detect stress focy. And on the step Cell Detection I faced such a problem "[Cell Centers]: Error:module 'cv2.cv2' has no attribute 'ximgproc'". This error appears on the cell center determination point. Any suggestion on how to fix it? Thanks in advance.

**Comment by lnilya** [comment]
Hi yeah the OpenCV libraries for python are a mysterious mess, they sporadically don't run on some computers after building it. 

Just to make sure: 
1. Are you using the demo data or have your own (if you have your own, please send me the files you use). 
2. Are you running windows or mac os? 

Usually the fix will be to reinstall the libraries and repackage the software. I did it a few times before for previous bugs of this count. Let me know about question #2 especially for this. 

Cheers
Ilya 

ps: just mail me at ilya.shabanov@vuw.ac.nz might be easier!

**Reply by USemaniuk** [reply]
Hi,

I hope you are doing well.

I sent you a letter (usemaniuk@gmail.com). But I suppose I might be in a spam folder. I added my image to my letter.

Just in case, I duplicate my answers:

My colleague and I use Windows and we both get this error. We use our data. But also tried to use demo data, same error. Also, the program does not recognize the tiff format of images, we used images in png format.

I tried reinstalling the library, but the error is still the same after reinstallation.

Sincerely,
Uliana
